cmake/OpenCVModule.cmake

```diff
@@ -1337,8 +1337,8 @@ function(ocv_add_samples)
         endif()
         add_dependencies(${parent_target} ${the_target})
 
-        if(WIN32)
-          install(TARGETS ${the_target} RUNTIME DESTINATION "samples/${module_id}" COMPONENT samples)
+        if(INSTALL_BIN_EXAMPLES)
+          install(TARGETS ${the_target} RUNTIME DESTINATION "${OPENCV_SAMPLES_BIN_INSTALL_PATH}/${module_id}" COMPONENT samples)
```

Comment:
Please put default value into "OpenCVInstallLayout.cmake" too.

Reply:
Do you mean for `OPENCV_SAMPLES_BIN_INSTALL_PATH`? It is already there, e.g. Unix https://github.com/opencv/opencv/blob/3b00ee2afb4176a5a45def7939b816d3a8914c00/cmake/OpenCVInstallLayout.cmake#L64